### PR TITLE
Single dbjson

### LINKED ISSUE #41: Show one Recipe
Show one Recipe while clicking on the title in the Recipe Page

### PR DESCRIPTION
**Pull request by boldCM**
Shows the Single Recipe Page with Data from Db.JSON (still local, sorry about that)
Also allows to click on a link to direct to that specific Recipe

### CODE REVIEW

File: client/src/api/connectJSON.js [hunk 1]
```diff
@@ -6,7 +6,9 @@ export async function getRecipies() {
 }
 
 export async function getRecipeByRecipeName(RecipeName) {
-  const response = await fetch(`http://localhost:8080/recipes/${RecipeName}`);
+  const response = await fetch(
+    `http://localhost:8080/recipes?RecipeName=${RecipeName}`
+  );
```
Comment (arassaria) [inline client/src/api/connectJSON.js]: You could think of using a proxy, so you could shorten this
```
const response = await fetch(
    `/recipes?RecipeName=${RecipeName}`
);
```

File: client/src/components/Recipe/RecipeIngredients.js [hunk 2]
```diff
@@ -9,20 +10,23 @@ const Ingredient = styled.ul`
   }
 `;
 
-const RecipeIngredients = () => {
+const RecipeIngredients = ({ Ingredients, id }) => {
+  console.log(Ingredients);
```
Comment (arassaria) [inline client/src/components/Recipe/RecipeIngredients.js:14]: console.logs are great for development, but not for production 😉

**Review by arassaria — COMMENTED** [review]
LGTM 🚀

File: client/src/api/connectJSON.js (hunk 1 above)
Comment (boldCM) [reply client/src/api/connectJSON.js]: thanks!!!:) :)
I'm right on it with the proxy! :)

File: client/src/components/Recipe/RecipeIngredients.js [hunk 3]
```diff
@@ -9,20 +10,23 @@ const Ingredient = styled.ul`
...
+          <li key={id}>{ingredient}</li>
+        ))}
       </Ingredient>
     </>
   );
 };
 
+RecipeIngredients.propTypes = {
+  Ingredients: PropTypes.array,
+  id: PropTypes.number,
```
Comment (mhstrkmp) [inline client/src/components/Recipe/RecipeIngredients.js:29]: If your components are ready you could consider to append `.isRequired` to your PropTypes

Explanation:
- https://blog.logrocket.com/validating-react-component-props-with-prop-types-ef14b29963fc/

**Review by mhstrkmp — APPROVED** [review]
Very clean code congrats 👏

**Review by SvLowman — APPROVED** [review]
lgtm 👍

File: client/src/components/Recipe/RecipeIngredients.js (hunk 3 above)
Comment (boldCM) [reply client/src/components/Recipe/RecipeIngredients.js:29]: thanks :)

**Pull request merged**